Print full stacktrace to console when an error occurs

harmbuisman opened the pull request:
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: fixes #9202 
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

bryevdv commented on bokeh/server/protocol_handler.py:
```diff
@@ -99,7 +99,7 @@ def handle(self, message, connection):
         except Exception as e:
-            log.error("error handling message %r: %r", message, e)
+            log.error("error handling message %r: %r", message, e, exc_info=1)
             log.debug("  message header %r content %r", message.header, message.content, exc_info=1)
```
I think the `exc_info=1` should now be removed from here, so it is not duplicated on the error and debug lines. Also, I think it would be good to stat in the error message that turning on debug output will show the entire message (then anyone who wants it has all the information they need right there)

bryevdv commented:
Thanks for the PR @harmbuisman I had a small request above. Also the CI is currently having issues due to an unrelated problem, we are working resolving it.

harmbuisman commented:
> Thanks for the PR @harmbuisman I had a small request above. Also the CI is currently having issues due to an unrelated problem, we are working resolving it.

@bryevdv good suggestion, I removed the redundant stacktrace, added a suggestion to run in debug modus and changed 1 to True so that the code is more readable.

I do wonder though if the printing of the message in the error log is not the same as the message.contents in the debug log. The only difference in my repro is that debug prints the header now, but the contents seem to be the same.

bryevdv commented:
@harmbuisman I think the repo must have changed at some point. The `content` is potentially very large, which was the idea behind separating things out. But if that's the way things now I'm not inclined to change it. Maybe just delete the `debug` statement altogether, and revert the log message to remove the note about `debug` What do you think?

harmbuisman commented:
> @harmbuisman I think the repo must have changed at some point. The `content` is potentially very large, which was the idea behind separating things out. But if that's the way things now I'm not inclined to change it. Maybe just delete the `debug` statement altogether, and revert the log message to remove the note about `debug` What do you think?

@bryevdv, yes, let's do that, that would be the cleanest solution. I adapted the code.

bryevdv commented:
Thanks @harmbuisman !

harmbuisman commented:
> Thanks @harmbuisman !

@bryevdv, you're welcome, it is nice to be able to contribute. Thanks for thinking along
